src/views/IndexView.jsx

```diff
@@ -70,7 +74,7 @@ export default IndexView;
 const styles = {
   statement: {
     textAlign: "center",
-    fontSize: "28px",
-    padding: "0 20px",
+    fontSize: "20px",
```

Comment:
can you change it to somwhere between 1 - 2 rem? That should make it so font becomes smaller on mobile